task1/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import json
+from selenium import webdriver
+
+@pytest.fixture()
+def get_driver():
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(5)
```

Comment:
How about we keep the duration in config?

Reply:
Moved to config